justfile

```diff
@@ -5,32 +5,43 @@ _default:
 
 # Install cargo plugins used by this project
```

Comment:
While I'd love if we used cargo here I think this comment is not accurate! Does `uv venv` install the dependencies as well?

Reply:
ha. I just fixed this and then came here and found your comment!